packages/beacon-node/test/e2e/network/reqresp.test.ts

```diff
@@ -84,53 +84,6 @@ function runTests({useWorker}: {useWorker: boolean}): void {
     return [netA, netB, await getPeerIdOf(netA), await getPeerIdOf(netB)];
   }
 
-  // it("should send/receive a ping message", async function () {
```

Comment:
Do you know why those were commented out in the first place and can be removed now?

Reply:
This was commented a year earlier in following PR. https://github.com/ChainSafe/lodestar/pull/5229/files#diff-ed1686203e4792b95a12ac893ea47ec4435a2c453ff81ccd14b0fa83943a82cc

I believe testing those messages are not necessary as we are testing connection and other messages already.